src/core/Rule.ml

```diff
@@ -547,6 +547,8 @@ and dependency_pattern = {
 }
 [@@deriving show, eq]
 
+type sca_mode = [ `SCA of dependency_formula ] [@@deriving show]
```

Comment:
What's SCA? The other modes don't use acronyms. If it's about dependencies, call it "dependency_mode" or something that describes what it is, nothing more, nothing else, because maybe it could be reused in other contexts than "SCA".